docs/configuration.md

```diff
@@ -221,6 +221,25 @@ of the most common options to set are:
     This option is currently supported on YARN and Kubernetes.
   </td>
 </tr>
+<tr>
+ <td><code>spark.executor.resource.{resourceType}.count</code></td>
+  <td>0</td>
+  <td>
+    The number of a particular resource type to use per executor process.
+    If this is used, you must also specify the
+    <code>spark.executor.resource.{resourceType}.discoveryScript</code>
```

Comment:
minor: I think a script should work. Should we also allow a class that implements an auto-discovery interface? Which one would be preferred by users? We don't need to support it in this PR.

Reply:
I think it would be easier for users to use the script, we can add in a class later if we need to.